config/plugins/interactive_environments/phinch/static/js/phinch.js

```diff
@@ -0,0 +1,44 @@
+function load_notebook(url){
+    $( document ).ready(function() {
+        test_ie_availability(url, function(){
+            append_notebook(url)
+        });
+    });
+}
+
+function keep_alive(notebook_access_url){
```

Comment:
@shiltemann I guess this function proofed to be useful in Jupyter. What do you think should we move it into the shared IE-JS file?